src/contact/containers/DisplayInfo.js

```diff
@@ -0,0 +1,21 @@
+import { connect } from 'react-redux';
+import DisplayInfo from '../components/DisplayInfo';
+import * as actions from '../actions/Actions';
+import { bindActionCreators } from 'redux';
+
+
+const mapStateToProps = (state) => {
+	return {
+		info : state.reducer.details,
```

Comment:
Why you are further nesting the props and creating new reference, this will mean you are unnecessary re-rendering your whole application for each and every change.

Reply:
please suggest the change here?